### DISCUSSION TITLE
List of classes and the correct includes for bootstrap 5.3

### DISCUSSION THREAD

**Discussion by archnakj** [twbs/bootstrap#39003] [Q&A]
Hello, Is there a master list of classes for bootstrap 5.3 available somewhere?

I was having trouble getting the navbar toggler icon to work as documented here: https://getbootstrap.com/docs/5.3/components/navbar/

Note that the output for some of the examples does not show the navbar-toggler-icon.

Upon pulling up code in Stackblitz, which does show the example working properly, I see an additional include:

https://getbootstrap.com/docs/5.3/assets/css/docs.css

This file is not readable (at least by me!). Should I be including this in my project? The main bootstrap page does not recommend this file be included.

Thanks!

**Comment by julien-deramond** [comment]
Hello @archnakj 

> Is there a master list of classes for bootstrap 5.3 available somewhere?

Unfortunately no, but I started https://github.com/twbs/bootstrap/pull/37076 a few months ago to include a glossary in the documentation. I hope that I'll be able to finish it one day.

> This file is not readable (at least by me!). Should I be including this in my project? The main bootstrap page does not recommend this file be included.

I confirm that you should omit this `docs.css` in your project. We include it in our StackBlitz examples because sometimes it's needed for some extra styles to display some icons or to render correctly some grid elements for instance. But for the navbars, you shouldn't need it. I tried a few examples in StackBlitz that seem to work correctly without this `docs.css` within StackBlitz; which ones are problematic for you?

**Comment by archnakj** [comment]
Hello, If you look at this link: https://getbootstrap.com/docs/5.3/components/navbar/

there are many examples that use the classes navbar-toggler and navbar-toggler-icon but the icon shows up in the displayed output only in three of them. If I pull up the examples in stackblitz, the icon does show up.

If putting together the glossary is something I could do (if you could tell me where to look to put it together), let me know. I am happy to help out.

Thanks very much for your help!
